src/admin-views/aggregator/origins/ical.php

```diff
@@ -32,27 +32,33 @@ class="tribe-ea-field tribe-ea-dropdown tribe-ea-size-large tribe-import-type"
 				data-hide-search
 				data-prevent-clear
 			>
-				<option value=""></option>
 				<option value="manual"><?php echo esc_html__( 'One-Time Import', 'the-events-calendar' ); ?></option>
 				<option value="schedule"><?php echo esc_html__( 'Scheduled Import', 'the-events-calendar' ); ?></option>
 			</select>
 		<?php endif; ?>
 
-		<select
-			name="aggregator[ical][import_frequency]"
-			id="tribe-ea-field-ical_import_frequency"
-			class="tribe-ea-field tribe-ea-dropdown tribe-ea-size-large tribe-dependent"
-			placeholder="<?php echo esc_attr( $frequency->placeholder ); ?>"
-			data-hide-search
+		<span
 			data-depends="#tribe-ea-field-ical_import_type"
-			data-condition="schedule"
-			data-prevent-clear
+		    data-condition="schedule"
```

Comment:
:no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 3dee321d6c4169a3583e5471b2449b6d</sub>